app/src/main/java/org/wikipedia/bridge/JavaScriptActionHandler.kt

```diff
@@ -31,10 +32,12 @@ object JavaScriptActionHandler {
     @JvmStatic
     fun setFooter(context: Context, model: PageViewModel): String {
         if (model.page == null) {
-            return "";
+            return ""
         }
 
-        val showLanguagesLink = model.page!!.pageProperties.languageCount > 0
+        val languageCount = L10nUtil.getUpdatedLanguageCountIfNeeded(model.page!!.title.wikiSite.languageCode(),
```

Comment:
Should take this `languageCount` to the `menuLanguagesTitle` as the correct value of language count?

Reply:
Doh!